firefox_puppeteer/tests/test_tabs.py

```diff
@@ -2,78 +2,152 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette import Wait
-from marionette.errors import NoSuchElementException
-
 from firefox_ui_harness.testcase import FirefoxTestCase
 
 
-class TestTabs(FirefoxTestCase):
+class TestTabBar(FirefoxTestCase):
+
+    def test_basics(self):
+        tabbar = self.browser.tabbar
+
+        self.assertEqual(tabbar.window, self.browser)
```

Comment:
Any ui library based on UIBaseLib will need a reference to the window it is contained in, e.g. to retrieve dtds and others. Given that we have wrappers to bring in additional features for windows, there has to be a need to reference the appropriate handle. If you have a proposal please file a bug with some examples, so I could understand what you mean. Thanks.